include/fastgltf/parser.hpp

```diff
@@ -265,6 +268,7 @@ namespace fastgltf {
         constexpr std::string_view KHR_materials_unlit = "KHR_materials_unlit";
         constexpr std::string_view KHR_materials_volume = "KHR_materials_volume";
         constexpr std::string_view KHR_mesh_quantization = "KHR_mesh_quantization";
+        constexpr std::string_view EXT_mesh_gpu_instancing = "EXT_mesh_gpu_instancing";
```

Comment:
why is this just put randomly in between the KHR extensions? these are sorted alphabetically.

Reply:
Ah, wrong gues, then, I thought it was about what it was related to (mesh, in this case)

Reply:
Fixed.